src/clib/pio_varm.c

```diff
@@ -64,7 +64,8 @@ int PIOc_put_varm (int ncid, int varid, const PIO_Offset start[], const PIO_Offs
                 *request = PIO_REQ_NULL;
             }
             vdesc->nreqs++;
-            flush_output_buffer(file, false, 0);
+            if (ierr == PIO_NOERR)
```

Comment:
I know the old PIO code omits braces when a single statement is enclosed in an if statement, But I would recommend adding the braces for clarity

Reply:
OK, I will add the braces.